WonderMake/Program/Program.cpp

```diff
@@ -33,6 +33,8 @@ void Program::StartNewFrame()
 	myCameraPtr->Update();
 
 	myRendererPtr->StartFrame();
+
+	myAudioManagerPtr->Update();
```

Comment:
Shouldn't this be in the Update function instead?

Reply:
Fixed